client-ui/src/app/components/modal/modal-expression/modal-expression.component.less

```diff
@@ -150,6 +149,25 @@
                 font-family: "Font Awesome 5 Free";
                 content: "\f02b";
                 color: darkcyan;
+                font-size: 12px;
+              }
+            }
+
+            &.expr-Alias {
```

Comment:
please, use dash-case all lower naming convention for CSS classes names. 

expr-Alias => expr-alias

Reply:
In this case `Alias` is a dynamic part of a style class name, which is taken from the `Expression._typeHint`. Ideally I would prefer to leave the original value (i.e. `expr.Alias`), but since dot is a reserved character CSS I had to escape it. `\.` notation doesn't seem to be a standard, so I replaced it with a dash doesn't belong to the value domain and couldn't create collisions. Yes I could replace camel case with a sneak case and lowercase it all, but the more changes you do to the value the more difficult it gets to look it up and distinguish later. 
That's why I opted to only do a single mandatory replacement `.`->`-`

Reply:
Oh, you are calculating a class name in TS and call it className, did not mention that before.

It is not a very good idea to do so.
But I do not know if I should comment it in that case as you kind of add some piece of mess to another pice of a mess, where most staff are of type any. It is kind of quick fix and, I guess, it is good enough for the current state.

will mark that conversation as resolved.